code/style.css

```diff
@@ -1,4 +1,755 @@
+/* GENERAL */
+
 html {
-  background: #0025ff;
-  color: #fff;
+  background: white;
+  color:black;
+  margin: 0px;
+  padding: 0px;
```

Comment:
When writing `margin: 0;` there is no need to write a unit such as px, rem etc after. Zero is always zero and doesn't need a unit ✌️